app/controllers/teachers/quizzes_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Teachers
+  class QuizzesController < BaseController
+    def index
+      @classroom = Classroom.find params[:classroom_id]
```

Comment:
You can omit this line if you use a `before_action` in the controller. See the `LearningModulesController`.